cirq-core/cirq/transformers/analytical_decompositions/two_qubit_to_cz.py

```diff
@@ -23,9 +23,9 @@
 
 from cirq import ops, linalg, protocols, circuits
 from cirq.transformers.analytical_decompositions import single_qubit_decompositions
+from cirq.transformers.eject_phased_paulis import eject_phased_paulis
```

Comment:
Module import causes problems in this case because `eject_phased_paulis` is the module name as well as the function name. I've followed this pattern for other imports from the transformer package as well, like `eject_z`. 

I can change it to a lazy import if you have strong preference.

Reply:
Ahhh, that's fine, no strong preference.